target/linux/ramips/files/drivers/net/ethernet/ralink/ethtool.c

```diff
@@ -159,11 +159,13 @@ static void fe_get_ringparam(struct net_device *dev,
 
 static void fe_get_strings(struct net_device *dev, u32 stringset, u8 *data)
 {
-	switch (stringset) {
-	case ETH_SS_STATS:
-		memcpy(data, *fe_gdma_str, sizeof(fe_gdma_str));
-		break;
-	}
+	int i;
+
+	if (stringset != ETH_SS_STATS)
+		return;
+
+	for (i = 0; i < ARRAY_SIZE(fe_gdma_str); i++)
+		ethtool_sprintf(&data, "%s", fe_gdma_str[i]);
```

Comment:
I would recommend avoiding such unrelated refactorings, because then its not "use ethtool_sprintf instead of memcpy" anymore, thus making the review harder. BTW if you look at `fe_get_sset_count()` you're actually introducing inconsistency, so I would keep it as it is.

Reply:
Upstream uses a combination of both styles. Probably better to reduce indentation.